test/environment_unittest.cc

```diff
@@ -1,5 +1,6 @@
 #include "../src/environment.h"
 #include "fake_http_server.h"
+#include "temp_file.h"
```

Comment:
Why do we include this twice? If you want to move it, please delete the `#include` in line 5.

Reply:
Done. (This was probably just due to a merge.)